### PR TITLE
Add planned enhancement: Edit command ui behaviour

### LINKED ISSUE #230: [PE-D][Tester B] Show all members and events after using edit command
**Steps discovering this bug:**

When I'm editing a member in this current list:

The command I enter: `editMember 1 t/neighbor`

After enter this command, the system display all the members and events, just like below:

- **Expect:** the edit command won't change the current display list.

- **Actual:** seems like edit command unintentionally call the command list to display all members and events.

Since UG didn't describe what the command edit will do, if your group intentionally designs command `edit` to functions as this, I guess that it is better to modify it a bit.

-------------
Labels: `type.FeatureFlaw` `severity.Medium`
original: KumChaiYin/ped#6

### PR DESCRIPTION
**Pull request by dylkaw**
Closes #230

What have you done in this PR?
* Added planned enhancement to prevent edit commands from returning to list view

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -435,6 +435,18 @@ due to the presence of the special character `'` in the event name.
...
 characters in the name field and not to be restricted to just alphanumeric characters.
 
+### Make UI stay on current view upon editMember or editEvent
+**Current Implementation:**
+* **Current Issue:** Currently, calling `editMember` or `editEvent` will result in the listing of all events and members.
+* **Example:** If the user is viewing a filtered list for example if they have just called `viewMember`, if the user
+  then edits a member or event using `editMember` or `editEvent`, all events and members will be displayed instead of their current
+filtered view. This will cause inconvenience for the user as they will have to relocate the edited member/event.
+
+**Proposed solution:**
```
Comment (SelwynAng) [inline docs/DeveloperGuide.md]: Suggestion to capitalise the 's' in solution

**Review by SelwynAng — APPROVED** [review]
LGTM aside from some formatting nitpicks

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (dylkaw) [reply docs/DeveloperGuide.md]: Done!

**Review by wr1159 — APPROVED** [review]
LGTM

**Pull request merged**